content/F2014/F14-media-sarah-sharp.md

```diff
@@ -219,30 +219,29 @@ for that because I negotiated with the professors so I could get credit.
 
 The lesson I took was basically rules can be bent.  Being able to talk with
 your professors and being able to get extra credit for cool projects or do
-projects with profs, that wasn't something I really thought of as this high
+projects with professors, that wasn't something I really thought of as this high
 school student from this really small town that's trying to follow the rules.
 It's fun to try to be able to find a project that you are passionate about and
 see if people will help you work on it.
 
-So in college, Professor Bart Massey beside helping me out with projects
-also tried to into introduce me to the local tech scene. In Portland, there
+So in college, Professor Bart Massey, beside helping me out with projects,
+also tried to introduce me to the local tech scene. In Portland, there
 is a large open source community.  And there are a bunch of Linux kernel
 hackers that are there.  Every month they get together and have kernel beerings
 where they have dinner at some particular restaurant and just sit around and
-talk.  So that was like, oh, you should come to these, Sarah.  You should meet
-these people.  I was kind of shy and said I don't know if I want to come, what
-if I don't drink, is that okay?  He said that's fine.  You should come anyway.
+talk.  So Bart was like, "Oh, you should come to these, Sarah.  You should meet
+these people."  I was kind of shy and said, "I don't know if I want to come, what
+if I don't drink, is that okay?"  He said that's fine.  You should come anyway.
```

Comment:
to be consistent, `that's fine. You should come anyway.` should probably be in quotes.